src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
I generally agree with Cyrus here. If I recall the low memory situation correctly, there's a specific problem (you have low memory) and a specific solution (we'll turn off global analysis). In this situation we don't know what's wrong and since it's an implementation detail, not a feature, it's not clear what we're going to be doing. I think we'd be better off saying something more general like what Cyrus suggested.